### PR TITLE
[zh-cn] Update `setup/_index.md`

### PR DESCRIPTION
**Pull request by yanrongshi**

### CODE REVIEW

**Comment by jihoon-seo** [comment]
The commit message is not mached with the change this PR suggests.

**Comment by yanrongshi** [comment]
> 提交消息与此 PR 建议的更改不匹配。

what's you mean? Can you be more specific? I'm new here. Not really. Thank you！

**Comment by jihoon-seo** [comment]
The commit message is `[zh-cn] update /create-cluster-kubeadm.md` (see [here](https://github.com/kubernetes/website/pull/34540/commits)),
but this PR suggests the change of `content/zh-cn/docs/setup/_index.md` file, not `create-cluster-kubeadm.md` file.

So it would be great to change the commit message to, for example,
```
[zh-cn] Update `setup/_index.md`
```

File: content/zh-cn/docs/setup/_index.md [hunk 1]
```diff
@@ -57,6 +57,18 @@ There are also other standardized and custom solutions across a wide range of cl
 bare metal environments.
 -->
 可以[下载 Kubernetes](/releases/download/)，在本地机器、云或你自己的数据中心上部署 Kubernetes 集群。
+
+<!-- Several [Kubernetes components](/docs/concepts/overview/components/) such as `kube-apiserver` or `kube-proxy` can also be
```
Comment (howieyuen) [inline content/zh-cn/docs/setup/_index.md:61]:
```suggestion
<!-- 
Several [Kubernetes components](/docs/concepts/overview/components/) such as `kube-apiserver` or `kube-proxy` can also be
```

File: content/zh-cn/docs/setup/_index.md [hunk 2]
```diff
@@ -57,6 +57,18 @@ There are also other standardized and custom solutions across a wide range of cl
...
+deployed as [container images](/releases/download/#container-images) within the cluster. -->
```
Comment (howieyuen) [inline content/zh-cn/docs/setup/_index.md:62]:
```suggestion
deployed as [container images](/releases/download/#container-images) within the cluster. 
-->
```

File: content/zh-cn/docs/setup/_index.md [hunk 3]
```diff
@@ -57,6 +57,18 @@ There are also other standardized and custom solutions across a wide range of cl
...
+一些 [Kubernetes 组件](/zh-cn/docs/concepts/overview/components/)，如 `kube-apiserver` 或 `kube-proxy`，
+也可以作为[容器镜像](/releases/download/#container-images)部署在集群中。
+
+<!-- It is **recommended** to run Kubernetes components as container images wherever
```
Comment (howieyuen) [inline content/zh-cn/docs/setup/_index.md:66]:
```suggestion
<!-- 
It is **recommended** to run Kubernetes components as container images wherever
```

File: content/zh-cn/docs/setup/_index.md [hunk 4]
```diff
@@ -57,6 +57,18 @@ There are also other standardized and custom solutions across a wide range of cl
...
+that is possible, and to have Kubernetes manage those components.
+Components that run containers - notably, the kubelet - can't be included in this category. -->
```
Comment (howieyuen) [inline content/zh-cn/docs/setup/_index.md:68]:
```suggestion
Components that run containers - notably, the kubelet - can't be included in this category. 
-->
```

File: content/zh-cn/docs/setup/_index.md [hunk 5]
```diff
@@ -57,6 +57,18 @@ There are also other standardized and custom solutions across a wide range of cl
...
+**建议**尽可能将 Kubernetes 组件作为容器镜像运行，并让 Kubernetes 管理这些组件。
+但是运行容器的组件——尤其是 kubelet，不能包含在这个类别中。
```
Comment (howieyuen) [inline content/zh-cn/docs/setup/_index.md:70]:
```suggestion
但是运行容器的组件 —— 尤其是 kubelet，不能包含在这个类别中。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Comment by tengqm** [comment]
Please squash your PR into one commit.

**Pull request closed without merging**